src/app/layout.tsx

```diff
@@ -26,13 +26,20 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-    <body className={cn(`${geistSans.variable} ${geistMono.variable}`,'font-sans grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-0' )}>
-    <Header/>
-    <main className="flex flex-col gap-8 row-start-2 items-center sm:items-start">
-      {children}
-    </main>
-    <Footer />
-    </body>
+      <body
+        className={cn(
+          `${geistSans.variable} ${geistMono.variable}`,
+          'grid grid-rows-[auto_1fr_auto] gap-16',
+          'items-center justify-items-center',
+          "bg-neutral-100 font-sans  min-h-screen  sm:p-0"
+        )}
+      >
+        <Header />
+        <main className="flex flex-col gap-8 row-start-2 items-center sm:items-start">
```

Comment:
don't style it unless this is the style for the whole project, reason being that if another page wants a different style we will be stuck with it, do this in the page.ts